test/fprime_gds/executables/test_data_product_writer.py

```diff
@@ -0,0 +1,28 @@
+#!/bin/env python3
+
+import subprocess
+import os
+import sys
+
+
+# Specify the directory where the .bin files are located
+directory = "./"
+
+# Loop through each file in the specified directory
+for filename in os.listdir(directory):
+    # Check if the file is a .bin file
+    if filename.endswith(".bin"):
+        # Construct the full path to the file
+        file_path = os.path.join(directory, filename)
+        
+        # Construct the command to run
+        command = f"../../../src/fprime_gds/executables/data_product_writer.py {file_path}"
+        
+        # Execute the command
+        process = subprocess.run(command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
Instead of calling the script directly, we should import the source module (we do that [here](https://github.com/nasa/fprime-gds/blob/devel/test/fprime_gds/executables/test_run_deployment.py#L7)) and call its main function